Dockerfile

```diff
@@ -109,3 +109,8 @@ RUN \
 
 # add local files
 COPY root/ /
+
+# healthcheck
+RUN chmod +x /healthcheck.sh
```

Comment:
This chmod is not needed if the file is already +x in the git repo

Reply:
This is true. I saw other `chmod` happening in the Dockerfile and thought that might have been preferred, but I'm happy to change the permissions on the file instead. My only comment against that is that file permissions in GitHub can be easy to overlook (in my opinion).

Reply:
not for us